app/models/card.rb

```diff
@@ -1,7 +1,21 @@
 class Card < ActiveRecord::Base
-  before_create :set_review
+  before_validation :set_review
+  validate :check_original_translated_equal
+  validates :original, presence: true
+  validates :translated, presence: true
+  validates :review, presence: true
 
-  def set_review
-    self.review = Time.now + 3.days
-  end
+  protected
+    def set_review
+      self.review = Time.now + 3.days
+    end
+
+    def check_original_translated_equal
+      self.original = original.mb_chars.downcase.to_s.squeeze(' ').lstrip
+      self.translated = translated.mb_chars.downcase.to_s.squeeze(' ').lstrip
+      if original == translated
+        errors.add(:original,
+                   ' = Translated. Вводимые значения должны отличаться.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.